frontend_tests/node_tests/people.js

```diff
@@ -159,6 +160,10 @@ run_test('basics', () => {
     assert.deepEqual(
         active_humans.sort((p) => p.user_id),
         [me, isaac]);
+    // Test has_only_one_admin function
```

Comment:
Would it be better to move this into a separate test? And also add a check for the `false` case?